.vscode/scripts/import-example-app.sh

```diff
@@ -27,10 +29,18 @@ else
   rm -rf app/
   cp -a $VELOCITAS_EXAMPLES_PATH/$CHOSEN_EXAMPLE/. app/
```

Comment:
should fix the ownership error
```suggestion
  cp -r $VELOCITAS_EXAMPLES_PATH/$CHOSEN_EXAMPLE/. app/
```